src/lib/env-vars.ts

```diff
@@ -76,12 +76,12 @@ export const SUPERVISOR_CONFIG_VAR_PROPERTIES: {
 	},
 	RESIN_SUPERVISOR_LOG_CONTROL: {
 		enum: ['false', 'true'],
-		description: 'Enable / Disable logs from being sent to Resin',
+		description: 'Enable / Disable logs from being sent to balena',
 		default: 'true',
 	},
 	RESIN_SUPERVISOR_POLL_INTERVAL: {
 		type: 'integer',
-		description: 'Define the Resin API Poll interval in milliseconds',
+		description: 'Define the balena API Poll interval in milliseconds',
```

Comment:
Might as well change `Poll` to `poll` while you're at it.

Reply:
^^ This. good spot man